src/components/RoomView/ResetTeams.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { ref, remove, set } from 'firebase/database';
+import { database } from '../../utils/firebase';
+import { useSelector } from 'react-redux';
+import { RootState } from '../../store';
+
+const ResetTeams = () => {
```

Comment:
because we are storing teamId on the player redux slice, we also need to make sure we dispatch a null to that.
Otherwise, say bob was team red, and we reset the teams, his chat box color will still be red.
i think some other logic uses this teamId on player, as well.

Reply:
this was a total oversight on my end! the new logic i introduced should fix it all, but let me know if you see any bugs/issues